Bottom: 2a52f91e2cf5d3de6d9d46a0ac979c54b55d3d99
Top:    9771ff8a3d26d35b52c2411516a08b33fefc3432
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-01-26 23:20:03 -0500

Refresh of jds-lirc-base

---

diff --git a/drivers/input/evdev.c b/drivers/input/evdev.c
index a651856..d601a3a 100644
--- a/drivers/input/evdev.c
+++ b/drivers/input/evdev.c
@@ -679,14 +679,6 @@ static long evdev_do_ioctl(struct file *file, unsigned int cmd,
 
 				return 0;
 			}
-
-			if (_IOC_NR(cmd) == _IOC_NR(EVIOIRSEND)) {
-
-				if (evdev_ir_send_from_user(p, _IOC_SIZE(cmd), &ir_command))
-					return -EFAULT;
-
-				return input_ir_send(dev, &ir_command, file);
-			}
 		}
 	}
 	return -EINVAL;
